main/tests/test_circle.py

```diff
@@ -0,0 +1,34 @@
+import pytest
+
+
+class TestCircle:
+
+    def test_get_coords(self):
+        circle = Circle()
+        assert circle.get_coords() == [0, 1, 2, 3, 4]
+
+    def test_set_coords(self):
+        circle = Circle([4, 3, 2, 1, 0])
+        circle.set_coords([0, 1, 2, 3, 4])
+
+        assert circle.get_coords() == [0, 1, 2, 3, 4]
+
+    def test_get_radius(self):
+        circle = Circle()
+        assert circle.get_radius() == 3
+

```

Comment:
Similar comment as the one above. I would initialize radius and coordinates in the Circle constructor.

Reply:
same with the other one